Cython/Compiler/Optimize.py

```diff
@@ -1990,6 +1990,7 @@ def _handle_simple_function_frozenset(self, node, pos_args):
         """
         if len(pos_args) != 1:
             return node
+        # TODO Doubt: What's the case for it
         if pos_args[0].is_sequence_constructor and not pos_args[0].args:
             del pos_args[0]
```

Comment:
BTW, for the constant literal case, it's probably worth changing sequence arguments into lists if they are tuples, to avoid creating a permanent tuple constant first that sticks around when all we need is the (deduplicated) frozenset constant.